frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.cy.ts

```diff
@@ -129,9 +129,7 @@ if (!Cypress.env('OPENSHIFT_CI') || Cypress.env('PLUGIN_PULL_SPEC')) {
               listPage.rows.shouldBeLoaded();
               listPage.filter.byName(PLUGIN_NAME);
               listPage.rows.shouldExist(PLUGIN_NAME);
-              if (!isLocalDevEnvironment && PLUGIN_PULL_SPEC) {
```

Comment:
Unrelated to the PR changes, but noticed this `if` is unnecessary since the parent block is already wrapped in this same `if`.